etc/profile-a-l/lobster.profile

```diff
@@ -30,7 +40,7 @@ noprinters
 notv
 
 disable-mnt
-private-bin curl,cut,fzf,grep,head,lobster,mv,patch,rm,sed,sh,tail,tput,tr,uname
+private-bin base64,bash,cat,curl,cut,date,dirname,echo,ffmpeg,ffprobe,find,fzf,grep,head,hxunent,lobster,ls,mkdir,mkfifo,nano,nohup,openssl,patch,pgrep,ps,rm,rofi,sed,sh,sleep,socat,tail,tee,tput,tr,ueberzugpp,uname,uuidgen,vim,wc
```

Comment:
I'm seeing two additional commands in [lobster](https://github.com/justchokingaround/lobster/blob/main/lobster.sh) that are missing from private-bin:
- command
- ln